### PR TITLE
Execute paypal without render

### PR DESCRIPTION
**Pull request by sfblaauw**

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Take a look to my comments on your previous pr.

**Comment by arkaitzgarro** [comment]
I am not sure, but I think that Paypal is expecting parameters being send by POST method, and in this way, we are sending them by GET. Have you tried if it really works this way?

https://www.paypal.com/cgi-bin/webscr?cmd=p/pdn/howto_checkout-outside

File: Controller/PaypalWebCheckoutController.php [hunk 1]
```diff
@@ -36,15 +37,20 @@ class PaypalWebCheckoutController extends Controller
...
+            ->add('cmd', 'hidden', array('empty_data' => '_cart'))
+            ->add('upload', 'hidden', array('empty_data' => 1))
+            ->add('lc', 'hidden', array('empty_data' => $request->getLocale()))
+        ;
+        $form->submit(null, true);
+
+        $action = $form->getConfig()->getAction();
+        $url = $action.'?'.http_build_query($form->getData());
+
+        $respone = RedirectResponse::create($url);
```
Comment (mmoreram) [inline Controller/PaypalWebCheckoutController.php]: $response

File: Controller/PaypalWebCheckoutController.php [hunk 2]
```diff
@@ -16,6 +16,7 @@
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\RedirectResponse;
```
Comment (mmoreram) [inline Controller/PaypalWebCheckoutController.php]: Not sorted. Please, use the php-formatter tool for normalizing this block

File: Controller/PaypalWebCheckoutController.php (hunk 1 above)
Comment (mickaelandrieu) [inline Controller/PaypalWebCheckoutController.php]: Yup. Take a look to our previous comments on the original pull request :)

**Comment by sfblaauw** [comment]
Hi everyone! Sorry but I will be very busy this month. I have to manage two new project. One of this project is a e-shop (I will do it with Elcodi, for sure) but until I start with the payment process I will be not able to work on this pull.
@arkaitzgarro yes, I tried it and works

**Comment by mickaelandrieu** [comment]
Ok, let us know when you can finish this we will merge into master branch.

**Comment by mickaelandrieu** [comment]
@sfblaauw hi ! can you give us a relative delay ? Maybe can you be available the last week ?

We like your work and this need a little improvement to be part of PaymentSuite and of elcodi's project ;)

**Comment by sfblaauw** [comment]
Hi @mickaelandrieu! I will not be available in the next two weeks, sorry. After I will work on it, I promise.

**Comment by mickaelandrieu** [comment]
Hi @sfblaauw, can you rebase the pull request and address the comments ? We need you :)

**Pull request closed without merging**